libr/bin/format/pe/pe.c

```diff
@@ -111,6 +110,15 @@ ut64 PE_(r_bin_pe_get_image_base)(struct PE_(r_bin_pe_obj_t)* bin) {
 	return imageBase;
 }
 
+static PE_DWord bin_pe_calc_alignment(RBinPEObj* bin, PE_DWord value) {
```

Comment:
that indentation hurts. sorry to merge without noticing this. gotta fix in master now